.github/workflows/brief.yml

```diff
@@ -0,0 +1,160 @@
+name: CMake and Test
+
+on: [push]
+
+env:
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    if: github.event.pull_request.draft == false # avoid triggering on draft
+    name:
+      ${{ matrix.platform }}, Parser option ${{ matrix.xml_parser_option }},
+      with namespaces ${{ matrix.with_namespace}}, strict includes ${{
+      matrix.strict }}, with examples ${{ matrix.with_examples}}, package option
+      ${{ matrix.package_option}}
+    strategy:
+      fail-fast: false
+      matrix:
+        xml_parser_option: ["-DWITH_LIBXML", "-DWITH_EXPAT", "-DWITH_XERCES"]
```

Comment:
To get the Expat and Xerces options building requires also to have `-DWITH_LIBXML=FALSE`, as LibXML is "on" be default and needs to be "switched off".

Reply:
yes, that is checked in the rules below and will be added